_tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

```diff
@@ -481,13 +481,15 @@ Request parameters | Description
 `partial` | Whether to allow the restoration of partial snapshots. Default is `false`.
 `rename_pattern` | If you want to rename indexes as you restore them, use this option to specify a regular expression that matches all indexes you want to restore. Use capture groups (`()`) to reuse portions of the index name.
 `rename_replacement` | If you want to rename indexes as you restore them, use this option to specify the replacement pattern. Use `$0` to include the entire matching index name, `$1` to include the content of the first capture group, and so on.
+`rename_alias_pattern` | If you want to rename aliases as you restore them, use this option to specify a regular expression that matches all aliases you want to restore. Use capture groups (`()`) to reuse portions of the alias name.
+`rename_alias_replacement` | If you want to rename aliases as you restore them, use this option to specify the replacement pattern. Use `$0` to include the entire matching alias name, `$1` to include the content of the first capture group, and so on.
 `index_settings` | If you want to change [index settings]({{site.url}}{{site.baseurl}}/im-plugin/index-settings/) applied during the restore operation, specify them here. You cannot change `index.number_of_shards`.
 `ignore_index_settings` | Rather than explicitly specifying new settings with `index_settings`, you can ignore certain index settings in the snapshot and use the cluster defaults applied during restore. You cannot ignore `index.number_of_shards`, `index.number_of_replicas`, or `index.auto_expand_replicas`.
 `storage_type` | `local` indicates that all snapshot metadata and index data will be downloaded to local storage. <br /><br > `remote_snapshot` indicates that snapshot metadata will be downloaded to the cluster, but the remote repository will remain the authoritative store of the index data. Data will be downloaded and cached as necessary to service queries. At least one node in the cluster must be configured with the [search role]({{site.url}}{{site.baseurl}}/security/access-control/users-roles/) in order to restore a snapshot using the type `remote_snapshot`. <br /><br > Defaults to `local`.
 
 ### Conflicts and compatibility
 
-One way to avoid naming conflicts when restoring indexes is to use the `rename_pattern` and `rename_replacement` options. You can then, if necessary, use the `_reindex` API to combine the two. The simpler way is to delete existing indexes prior to restoring from a snapshot.
+One way to avoid indexes naming conflicts when restoring indexes is to use the `rename_pattern` and `rename_replacement` options. You can then, if necessary, use the `_reindex` API to combine the two. Simlarly, to avoid aliases naming conflicts when restoring index with aliases, you can use the `rename_alias_pattern` and `rename_alias_replacement` options. The simpler way is to delete existing indexes prior to restoring from a snapshot. 
```

Comment:
```suggestion
One way to avoid index naming conflicts when restoring indexes is to use the `rename_pattern` and `rename_replacement` options. You can then, if necessary, use the `_reindex` API to combine the two. However, if might be simpler to delete the existing indexes that caused the conflict prior to restoring from a snapshot.

Similarly, to avoid alias naming conflicts when restoring index with aliases, you can use the `rename_alias_pattern` and `rename_alias_replacement` options. 
```

Reply:
Updated